tests/cancun/eip6780_selfdestruct/test_selfdestruct_revert.py

```diff
@@ -392,3 +511,44 @@ def test_selfdestruct_not_created_in_same_tx_with_revert(
     )
 
     state_test(env=env, pre=pre, post=post, tx=tx)
+
+
+@pytest.mark.valid_from("Cancun")
+def test_coverage_script_fix(
+    state_test: StateTestFiller,
+    env: Environment,
+    pre: Alloc,
+):
+    """Fixes coverage script by making use of SWAP3 and JUMP."""
```

Comment:
@marioevz I know that we've added such "coverage fixer tests" in the past. What's your take here?

I would lean towards bypassing CI checks for squash and merge and avoid unnecessary tests.